R/utils.R

```diff
@@ -210,3 +208,78 @@ convert <- function(x) {
   x[y]      <- converted
   x
 }
+
+
+#' Convert Redcap data dictionary into {matchmaker} dictionary format
+#'
+#' @param metadata a `data frame` with the data dictionary associated to a
+#'    Redcap project
+#' @param field_column the name of the column, in the input dictionary, that
+#'    contains the field names in of the Redcap project data
+#' @param opt_column the name of the column, in the input dictionary, that
+#'    contains the definition of the choices in every column of the Redcap
+#'    project data
+#'
```

Comment:
It would be useful to include an example of the metadata `data frame`  and the resulting {matchmaker} dictionary format